src/coreclr/src/jit/jitconfigvalues.h

```diff
@@ -286,6 +286,14 @@ CONFIG_INTEGER(JitDisableSimdVN, W("JitDisableSimdVN"), 0) // Default 0, ValueNu
                                                            // If 3, disable both SIMD and HW Intrinsic nodes
 #endif                                                     // FEATURE_SIMD
 
+// Default 0, enable the CSE of Constants, including nearby offsets. (only for ARM64)
+// If 1, disable all the CSE of Constants
+// If 2, enable the CSE of Constants but don't combine with nearby offsets. (only for ARM64)
+// If 3, enable the CSE of Constants including nearby offsets. (all platforms)
```

Comment:
> (all platforms) [](start = 62, length = 16)

just for completion, add ` (all platforms)`

Reply:
Yes, I have disable CSE of constants by default for non-ARM64 platforms.

see optcse.cpp lines 726-742

```
    bool disableConstCSE = false;

    int configValue = JitConfig.JitDisableConstCSE();

    // all platforms - disable CSE of constant values when config is 1
    if (configValue == 1)
    {
        disableConstCSE = true;
    }
#if !defined(TARGET_ARM64)
    // non-ARM64 platforms - also disable CSE of constant values when config is 0 or 2
    if ((configValue == 0) || (configValue == 2))
    {
        disableConstCSE = true;
    }
#endif
```